sdk/src/CrashReporter.ts

```diff
@@ -209,12 +205,12 @@ export default class CrashReporter {
     const symbolicateStackTrace = require('react-native/Libraries/Core/Devtools/symbolicateStackTrace');
     const cleanedStackFrames: StackFrame[] = __DEV__
       ? await symbolicateStackTrace(stackFrames)
-      : { stack: cleanFilePath(stackFrames) };
+      : {stack: cleanFilePath(stackFrames)};
 
     const stack = cleanedStackFrames || [].filter(filterOutReactFrames).map(noAddressAt);
 
     if (isFatal) {
-      this.tags.add('UnhandledError');
+      setCurrentTags(getCurrentTags().concat('UnhandledError'))
     }
```

Comment:
Add "Unhandled error" tag whenever an error enters this method but also append "Fatal" if it is a fatal error.